Created the Option to add one line per task for Time php

pstructures opened the pull request:
# New Option to add one line per task to generate invoice mass action
Added the option to add one line per task alongside the addonelineperuser section in time.php

Commit:
Add option to show one line per timespent entry

Revised changes with new development branch code

stickler-ci committed:
Fixing style errors.

Commit:
Update time.php

added the option to generate an invoice with one line per task

stickler-ci committed:
Fixing style errors.

eldy commented:
There is 1 conflict on file that can't be merged automatically.
Can you try to solve it (code must be inserted into last develop version) to be mergeable.

pstructures commented:
Hi Eldy, 

I have fixed the issues, as it was on an old version of my fork but have now fixed the issue.  Please see the new pull request from me.  

Matt

pstructures commented:
Already been done in the development branch
